pandas/tests/computation/test_eval.py

```diff
@@ -1890,6 +1890,14 @@ def test_negate_lt_eq_le(engine, parser):
         tm.assert_frame_equal(result, expected)
 
 
+def test_set_inplace():
+    # GH 47449
+    df = DataFrame({"A": range(1, 6), "B": range(10, 0, -2), "C": range(11, 16)})
+    expected = Series([21, 20, 19, 18, 17], index=[0, 1, 2, 3, 4], name="A")
+    df.eval("A = B + C", inplace=True)
+    tm.assert_series_equal(df["A"], expected)
```

Comment:
we need to test that a view on the DataFrame is updated. Can you replicate the code sample in the issue OP. i.e. assign a column to a variable (creating a view) before the `eval` method call. (similar to tests added in #47327)

Reply:
done